src/app/home/index.njk

```diff
@@ -17,16 +17,17 @@
 {% block content %}
 <div class="govuk-grid-row">
 	<div class="govuk-grid-column-two-thirds">
-    {% if userSession.StatusChangedTimestamp%}
+    {% if userSession.StatusChangedTimestamp %}
       <h1 class="govuk-heading-m">
         <span class="heading-secondary govuk-hint">
-          You last logged in on {{userSession.StatusChangedTimestamp}}
+          {% set timestamp = userSession.StatusChangedTimestamp.toString() %}
```

Comment:
i18n needed the timestamp as a String, hence the toString() call on what should be a Date object